<html>
<head>
<title>TestNG:  GRP-SI - WF 30,SI - WF 28,SI - WF 29,SI - WF 27</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>GRP-SI - WF 30,SI - WF 28,SI - WF 29,SI - WF 27</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>1/2/0</td>
</tr><tr>
<td>Started on:</td><td>Wed Jul 13 12:27:50 IST 2016</td>
</tr>
<tr><td>Total time:</td><td>322 seconds (322338 ms)</td>
</tr><tr>
<td>Included groups:</td><td>SI - WF 30 SI - WF 28 SI - WF 29 SI - WF 27 </td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.tests.WorkFlow.SI_WF27_CustomList_MultipleList_ListNames_DefaultCat()'><b>SI_WF27_CustomList_MultipleList_ListNames_DefaultCat</b><br>Test class: com.tests.WorkFlow<br>Test method: SI - WF 27-Custom List + Select Multiple List +List Names+Default Cat<br/>
<a href="#Output-1497377679" onClick='toggleBox("Output-1497377679", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1497377679" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1497377679">
[12:28:11] : <b><FONT SIZE="2">Launched Sysco UOM</FONT></b><br><br/>
[12:28:16] : <b><FONT SIZE="2">Login page verification :Pass</FONT></b><br><br/>
[12:28:16] : <b><FONT SIZE="2">Save username verification :Pass</FONT></b><br><br/>
[12:28:16] : <b><FONT SIZE="2">Sign In</FONT></b><br><br/>
[12:28:41] : <b><FONT SIZE="2">Sign in :Pass</FONT></b><br><br/>
[12:28:41] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Navigating to Setup inventory page :Pass</FONT></FONT></b><br><br/>
[12:28:43] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on skip :Pass</FONT></FONT></b><br><br/>
[12:28:43] : <b><FONT SIZE="2">Selecting custom list  </FONT></b><br><br/>
[12:28:43] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Selected  custom list  from SetupInventoryImportItems:Pass</FONT></FONT></b><br><br/>
[12:28:44] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Continue :Pass</FONT></FONT></b><br><br/>
[12:28:44] : <b><FONT SIZE="2">Selecting list</FONT></b><br><br/>
[12:28:49] : <b><FONT COLOR="GREEN"><FONT SIZE="2">List is selected :Pass</FONT></FONT></b><br><br/>
[12:28:49] : <b><FONT SIZE="2">Selecting list</FONT></b><br><br/>
[12:28:54] : <b><FONT COLOR="GREEN"><FONT SIZE="2">List is selected :Pass</FONT></FONT></b><br><br/>
[12:28:56] : <b><FONT SIZE="2">Selecting custom category from Locations page</FONT></b><br><br/>
[12:28:57] : <b><FONT COLOR="GREEN"><FONT SIZE="2">List Names selected from locations page :Pass</FONT></FONT></b><br><br/>
[12:28:57] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Continue :Pass</FONT></FONT></b><br><br/>
[12:28:57] : <b><FONT SIZE="2">Selecting default category from Food cost page</FONT></b><br><br/>
[12:29:05] : <b><FONT COLOR="GREEN"><FONT SIZE="2">default Category selected :Pass</FONT></FONT></b><br><br/>
[12:29:06] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Complete :Pass</FONT></FONT></b><br><br/>
[12:29:11] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on TakeHome :Pass</FONT></FONT></b><br><br/>
[12:29:17] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped  TrackInventory from inventory tools page:Pass</FONT></FONT></b><br><br/>
[12:29:17] : <b><FONT SIZE="2">Selecting location</FONT></b><br><br/>
[12:29:19] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Location is selected for item :Pass</FONT></FONT></b><br><br/>
[12:29:19] : <b><FONT SIZE="2">locationItem verify </FONT></b><br><br/>
[12:29:28] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Item verification done  :Fail</FONT></FONT></b><br><br/>
<b><font color='RED'>ERROR	:- expected [true] but found [false]</font></b><br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.components.pages.LocationsPage.VerifyListItemsOnLocation(LocationsPage.java:1823)
	at com.tests.WorkFlow.SI_WF27_CustomList_MultipleList_ListNames_DefaultCat(WorkFlow.java:2086)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
</pre></div><a href='#' onClick='toggleBox("stack-trace1497377679", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1497377679'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.components.pages.LocationsPage.VerifyListItemsOnLocation(LocationsPage.java:1823)
	at com.tests.WorkFlow.SI_WF27_CustomList_MultipleList_ListNames_DefaultCat(WorkFlow.java:2086)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
</pre></div></td>
<td>97</td>
<td>com.tests.WorkFlow@4667ae56</td></tr>
<tr>
<td title='com.tests.WorkFlow.SI_WF28_CustomList_MultipleList_ListNames_Customcategory()'><b>SI_WF28_CustomList_MultipleList_ListNames_Customcategory</b><br>Test class: com.tests.WorkFlow<br>Test method: SI - WF 28-Custom List + Select Multiple List +List Names as location+custom Cat<br/>
<a href="#Output-1330247343" onClick='toggleBox("Output-1330247343", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1330247343" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1330247343">
[12:29:47] : <b><FONT SIZE="2">Launched Sysco UOM</FONT></b><br><br/>
[12:29:53] : <b><FONT SIZE="2">Login page verification :Pass</FONT></b><br><br/>
[12:29:53] : <b><FONT SIZE="2">Save username verification :Pass</FONT></b><br><br/>
[12:29:53] : <b><FONT SIZE="2">Sign In</FONT></b><br><br/>
[12:30:17] : <b><FONT SIZE="2">Sign in :Pass</FONT></b><br><br/>
[12:30:17] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Navigating to Setup inventory page :Pass</FONT></FONT></b><br><br/>
[12:30:19] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on skip :Pass</FONT></FONT></b><br><br/>
[12:30:19] : <b><FONT SIZE="2">Selecting custom list  </FONT></b><br><br/>
[12:30:19] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Selected  custom list  from SetupInventoryImportItems:Pass</FONT></FONT></b><br><br/>
[12:30:19] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Continue :Pass</FONT></FONT></b><br><br/>
[12:30:19] : <b><FONT SIZE="2">Selecting list</FONT></b><br><br/>
[12:30:24] : <b><FONT COLOR="GREEN"><FONT SIZE="2">List is selected :Pass</FONT></FONT></b><br><br/>
[12:30:24] : <b><FONT SIZE="2">Selecting list</FONT></b><br><br/>
[12:30:30] : <b><FONT COLOR="GREEN"><FONT SIZE="2">List is selected :Pass</FONT></FONT></b><br><br/>
[12:30:32] : <b><FONT SIZE="2">Selecting custom category from Locations page</FONT></b><br><br/>
[12:30:32] : <b><FONT COLOR="GREEN"><FONT SIZE="2">List Names selected from locations page :Pass</FONT></FONT></b><br><br/>
[12:30:33] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Continue :Pass</FONT></FONT></b><br><br/>
[12:30:41] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Custom Category selected from Food cost page :Pass</FONT></FONT></b><br><br/>
[12:30:42] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Continue :Pass</FONT></FONT></b><br><br/>
[12:30:42] : <b><FONT SIZE="2">Creating multiple custom category</FONT></b><br><br/>
[12:30:48] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Custom Categories created:Pass</FONT></FONT></b><br><br/>
[12:30:57] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Next : Pass</FONT></FONT></b><br><br/>
[12:31:02] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Multiple items selected :Pass</FONT></FONT></b><br><br/>
[12:31:10] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Next : Pass</FONT></FONT></b><br><br/>
[12:31:19] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on done:Pass</FONT></FONT></b><br><br/>
[12:31:25] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on TakeHome :Pass</FONT></FONT></b><br><br/>
[12:31:31] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped  TrackInventory from inventory tools page:Pass</FONT></FONT></b><br><br/>
[12:31:31] : <b><FONT SIZE="2">Selecting location</FONT></b><br><br/>
[12:31:33] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Location is selected for item :Pass</FONT></FONT></b><br><br/>
[12:31:33] : <b><FONT SIZE="2">locationItem verify </FONT></b><br><br/>
[12:31:42] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Item verification done  :Fail</FONT></FONT></b><br><br/>
<b><font color='RED'>ERROR	:- expected [true] but found [false]</font></b><br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.components.pages.LocationsPage.VerifyListItemsOnLocation(LocationsPage.java:1823)
	at com.tests.WorkFlow.SI_WF28_CustomList_MultipleList_ListNames_Customcategory(WorkFlow.java:2160)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
</pre></div><a href='#' onClick='toggleBox("stack-trace1330247343", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1330247343'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.components.pages.LocationsPage.VerifyListItemsOnLocation(LocationsPage.java:1823)
	at com.tests.WorkFlow.SI_WF28_CustomList_MultipleList_ListNames_Customcategory(WorkFlow.java:2160)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
</pre></div></td>
<td>128</td>
<td>com.tests.WorkFlow@4667ae56</td></tr>
<tr>
<td title='com.tests.WorkFlow.SI_WF30_CustomListSingleList_CustomCategoryOnLocationsPage_customcategory()'><b>SI_WF30_CustomListSingleList_CustomCategoryOnLocationsPage_customcategory</b><br>Test class: com.tests.WorkFlow<br>Test method: SI - WF 30-Custom List + Select single  List +Custom Categories as location+Custom Cat<br/>
<a href="#Output-504582810" onClick='toggleBox("Output-504582810", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-504582810" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-504582810">
<b><font color='RED'>ERROR	:- Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.51.0', revision: '1af067d', time: '2016-02-05 19:15:17'
System info: host: 'tvmatp312483m.ad.infosys.com', ip: '10.155.162.74', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.8.0_40'
Driver info: driver.version: IOSDriver</font></b><br/>
</div>
</td>
<td><div><pre>org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &apos;2.51.0&apos;, revision: &apos;1af067d&apos;, time: &apos;2016-02-05 19:15:17&apos;
System info: host: &apos;tvmatp312483m.ad.infosys.com&apos;, ip: &apos;10.155.162.74&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.4&apos;, java.version: &apos;1.8.0_40&apos;
Driver info: driver.version: IOSDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:144)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:160)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:168)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:56)
	at zRfghY.t.b(WebDriverInstanceGenerator.java:86)
	at zRfghY.s.a(WebDriverFactory.java:36)
	at zRfghY.p.&lt;init&gt;(ThreadFactory.java:101)
	at zRfghY.p$1.c(ThreadFactory.java:97)
	at zRfghY.p$1.initialValue(ThreadFactory.java:95)
	at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
	at java.lang.ThreadLocal.get(ThreadLocal.java:170)
	at zRfghY.p.b(ThreadFactory.java:165)
	at com.iwaf.framework.testng.TestNGTestListener.onTestStart(TestNGTestListener.java:349)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1645)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:621)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: org.openqa.selenium.WebDriverException: org.apache.http.conn.HttpHostConnectException: Connect to 10.155.162.74:4723 [/10.155.162.74] failed: Connection refused
Build info: version: &apos;2.51.0&apos;, revision: &apos;1af067d&apos;, time: &apos;2016-02-05 19:15:17&apos;
System info: host: &apos;tvmatp312483m.ad.infosys.com&apos;, ip: &apos;10.155.162.74&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.4&apos;, java.version: &apos;1.8.0_40&apos;
Driver info: driver.version: IOSDriver
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:64)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
	... 41 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 10.155.162.74:4723 [/10.155.162.74] failed: Connection refused
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:162)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:52)
	... 42 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 55 more
</pre></div><a href='#' onClick='toggleBox("stack-trace504582810", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace504582810'><pre>org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &apos;2.51.0&apos;, revision: &apos;1af067d&apos;, time: &apos;2016-02-05 19:15:17&apos;
System info: host: &apos;tvmatp312483m.ad.infosys.com&apos;, ip: &apos;10.155.162.74&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.4&apos;, java.version: &apos;1.8.0_40&apos;
Driver info: driver.version: IOSDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:144)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:160)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:168)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:56)
	at zRfghY.t.b(WebDriverInstanceGenerator.java:86)
	at zRfghY.s.a(WebDriverFactory.java:36)
	at zRfghY.p.&lt;init&gt;(ThreadFactory.java:101)
	at zRfghY.p$1.c(ThreadFactory.java:97)
	at zRfghY.p$1.initialValue(ThreadFactory.java:95)
	at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
	at java.lang.ThreadLocal.get(ThreadLocal.java:170)
	at zRfghY.p.b(ThreadFactory.java:165)
	at com.iwaf.framework.testng.TestNGTestListener.onTestStart(TestNGTestListener.java:349)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1645)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:621)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: org.openqa.selenium.WebDriverException: org.apache.http.conn.HttpHostConnectException: Connect to 10.155.162.74:4723 [/10.155.162.74] failed: Connection refused
Build info: version: &apos;2.51.0&apos;, revision: &apos;1af067d&apos;, time: &apos;2016-02-05 19:15:17&apos;
System info: host: &apos;tvmatp312483m.ad.infosys.com&apos;, ip: &apos;10.155.162.74&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.4&apos;, java.version: &apos;1.8.0_40&apos;
Driver info: driver.version: IOSDriver
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:64)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
	... 41 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 10.155.162.74:4723 [/10.155.162.74] failed: Connection refused
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:162)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:52)
	... 42 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 55 more
</pre></div></td>
<td>5</td>
<td>com.tests.WorkFlow@4667ae56</td></tr>
<tr>
<td title='com.tests.WorkFlow.SI_WF30_CustomListSingleList_CustomCategoryOnLocationsPage_customcategory()'><b>SI_WF30_CustomListSingleList_CustomCategoryOnLocationsPage_customcategory</b><br>Test class: com.tests.WorkFlow<br>Test method: SI - WF 30-Custom List + Select single  List +Custom Categories as location+Custom Cat<br/>
<a href="#Output-153646652" onClick='toggleBox("Output-153646652", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-153646652" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-153646652">
<b><font color='RED'>ERROR	:- Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.51.0', revision: '1af067d', time: '2016-02-05 19:15:17'
System info: host: 'tvmatp312483m.ad.infosys.com', ip: '10.155.162.74', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.8.0_40'
Driver info: driver.version: IOSDriver</font></b><br/>
</div>
</td>
<td><div><pre>org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &apos;2.51.0&apos;, revision: &apos;1af067d&apos;, time: &apos;2016-02-05 19:15:17&apos;
System info: host: &apos;tvmatp312483m.ad.infosys.com&apos;, ip: &apos;10.155.162.74&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.4&apos;, java.version: &apos;1.8.0_40&apos;
Driver info: driver.version: IOSDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:144)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:160)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:168)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:56)
	at zRfghY.t.b(WebDriverInstanceGenerator.java:86)
	at zRfghY.s.a(WebDriverFactory.java:36)
	at zRfghY.p.&lt;init&gt;(ThreadFactory.java:101)
	at zRfghY.p$1.c(ThreadFactory.java:97)
	at zRfghY.p$1.initialValue(ThreadFactory.java:95)
	at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
	at java.lang.ThreadLocal.get(ThreadLocal.java:170)
	at zRfghY.p.b(ThreadFactory.java:165)
	at com.iwaf.framework.testng.TestNGTestListener.onTestFailure(TestNGTestListener.java:254)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1661)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1645)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:703)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: org.openqa.selenium.WebDriverException: org.apache.http.conn.HttpHostConnectException: Connect to 10.155.162.74:4723 [/10.155.162.74] failed: Connection refused
Build info: version: &apos;2.51.0&apos;, revision: &apos;1af067d&apos;, time: &apos;2016-02-05 19:15:17&apos;
System info: host: &apos;tvmatp312483m.ad.infosys.com&apos;, ip: &apos;10.155.162.74&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.4&apos;, java.version: &apos;1.8.0_40&apos;
Driver info: driver.version: IOSDriver
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:64)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
	... 41 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 10.155.162.74:4723 [/10.155.162.74] failed: Connection refused
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:52)
	... 42 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 55 more
</pre></div><a href='#' onClick='toggleBox("stack-trace153646652", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace153646652'><pre>org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &apos;2.51.0&apos;, revision: &apos;1af067d&apos;, time: &apos;2016-02-05 19:15:17&apos;
System info: host: &apos;tvmatp312483m.ad.infosys.com&apos;, ip: &apos;10.155.162.74&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.4&apos;, java.version: &apos;1.8.0_40&apos;
Driver info: driver.version: IOSDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:144)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:160)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:168)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:56)
	at zRfghY.t.b(WebDriverInstanceGenerator.java:86)
	at zRfghY.s.a(WebDriverFactory.java:36)
	at zRfghY.p.&lt;init&gt;(ThreadFactory.java:101)
	at zRfghY.p$1.c(ThreadFactory.java:97)
	at zRfghY.p$1.initialValue(ThreadFactory.java:95)
	at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
	at java.lang.ThreadLocal.get(ThreadLocal.java:170)
	at zRfghY.p.b(ThreadFactory.java:165)
	at com.iwaf.framework.testng.TestNGTestListener.onTestFailure(TestNGTestListener.java:254)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1661)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1645)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:703)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: org.openqa.selenium.WebDriverException: org.apache.http.conn.HttpHostConnectException: Connect to 10.155.162.74:4723 [/10.155.162.74] failed: Connection refused
Build info: version: &apos;2.51.0&apos;, revision: &apos;1af067d&apos;, time: &apos;2016-02-05 19:15:17&apos;
System info: host: &apos;tvmatp312483m.ad.infosys.com&apos;, ip: &apos;10.155.162.74&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.4&apos;, java.version: &apos;1.8.0_40&apos;
Driver info: driver.version: IOSDriver
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:64)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
	... 41 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 10.155.162.74:4723 [/10.155.162.74] failed: Connection refused
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:52)
	... 42 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 55 more
</pre></div></td>
<td>5</td>
<td>com.tests.WorkFlow@4667ae56</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.tests.WorkFlow.SI_WF29_CustomListSingleList_CustomCategoryOnLocationsPage_defaultcategory()'><b>SI_WF29_CustomListSingleList_CustomCategoryOnLocationsPage_defaultcategory</b><br>Test class: com.tests.WorkFlow<br>Test method: SI - WF 29-Custom List + Select single  List +Custom categories as location+Default Cat<br/>
<a href="#Output-333392524" onClick='toggleBox("Output-333392524", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-333392524" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-333392524">
[12:32:01] : <b><FONT SIZE="2">Launched Sysco UOM</FONT></b><br><br/>
[12:32:06] : <b><FONT SIZE="2">Login page verification :Pass</FONT></b><br><br/>
[12:32:07] : <b><FONT SIZE="2">Save username verification :Pass</FONT></b><br><br/>
[12:32:07] : <b><FONT SIZE="2">Sign In</FONT></b><br><br/>
[12:32:31] : <b><FONT SIZE="2">Sign in :Pass</FONT></b><br><br/>
[12:32:32] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Navigating to Setup inventory page :Pass</FONT></FONT></b><br><br/>
[12:32:34] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on skip :Pass</FONT></FONT></b><br><br/>
[12:32:34] : <b><FONT SIZE="2">Selecting custom list  </FONT></b><br><br/>
[12:32:34] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Selected  custom list  from SetupInventoryImportItems:Pass</FONT></FONT></b><br><br/>
[12:32:34] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Continue :Pass</FONT></FONT></b><br><br/>
[12:32:34] : <b><FONT SIZE="2">Selecting list</FONT></b><br><br/>
[12:32:40] : <b><FONT COLOR="GREEN"><FONT SIZE="2">List is selected :Pass</FONT></FONT></b><br><br/>
[12:32:42] : <b><FONT SIZE="2">Selecting custom category from Locations page</FONT></b><br><br/>
[12:32:42] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Custom category selected from locations page :Pass</FONT></FONT></b><br><br/>
[12:32:43] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Continue :Pass</FONT></FONT></b><br><br/>
[12:32:43] : <b><FONT SIZE="2">Selecting default category from Food cost page</FONT></b><br><br/>
[12:32:50] : <b><FONT COLOR="GREEN"><FONT SIZE="2">default Category selected :Pass</FONT></FONT></b><br><br/>
[12:32:51] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on Complete :Pass</FONT></FONT></b><br><br/>
[12:32:57] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped on TakeHome :Pass</FONT></FONT></b><br><br/>
[12:33:03] : <b><FONT COLOR="GREEN"><FONT SIZE="2">Tapped  TrackInventory from inventory tools page:Pass</FONT></FONT></b><br><br/>
Test completed : SI_WF29_CustomListSingleList_CustomCategoryOnLocationsPage_defaultcategory<br/>
</div>
</td>
<td></td>
<td>75</td>
<td>com.tests.WorkFlow@4667ae56</td></tr>
</table><p>
</body>
</html>